Exercises/NoModules/Chapter 19/Soln19_03/Soln19_03.cpp

```diff
@@ -0,0 +1,61 @@
+// Exercise 19-3. Comparing two sorting algorithms
```

Comment:
The quicksort values are not really close to the theoretical values. Example:
Number of comparisons for 4000 elements:
 - quicksort: 113047 (n*log(n): 47863)
Off by more than a factor 2.

Reply:
Interesting. Wonder why. A proper quicksort should, on average, perform at a ratio of 1.39. Added a ratio to the output and some code comments to acknowledge this apparent flaw with our implementation, as well as a bonus comparison with std::ranges::sort() (which of course performs flawlessly).